extensions/amp-story/amp-story.md

```diff
@@ -31,21 +31,21 @@ limitations under the License.
   </tr>
   <tr>
     <td width="40%"><strong>Availability</strong></td>
-    <td><div><a href="https://www.ampproject.org/docs/reference/experimental.html">Experimental</a></td>
+    <td><div><a href="https://amp.dev/documentation/guides-and-tutorials/learn/experimental.html">Experimental</a></td>
```

Comment:
Use the canonical url https://amp.dev/documentation/guides-and-tutorials/learn/experimental

Reply:
fixed